prototype-implementation/frontend-api/src/org/jetbrains/amper/frontend/parts.kt

```diff
@@ -154,6 +154,58 @@ data class PublicationPart(
         PublicationPart(group ?: "org.example", version ?: "SNAPSHOT-1.0")
 }
 
+data class KoverPart(val enabled: Boolean?): FragmentPart<KoverPart> {
```

Comment:
It's more like preference than a critical thing, but anyway:
I didn't get it why just don't have xml and html configs in `KoverPart` data class itself? 
```
data class KoverPart(val enabled: Boolean?, val html: KoverHtml?, val xml: KoverXml?): FragmentPart<KoverPart> {
```
I don't see why it's more convenient to have separate fragment parts
I also think it doesn't make sense to have for instance `KoverHtmlPart` without `KoverPart`

Reply:
I agree, and I'd considered doing it this way, but there were no examples of nested classes yet, and having a KoverHtmlPart allowed for an early idea of having koverHtml as a root setting, but I eventually figured that it's better to avoid adding more root settings instead of nesting in the kover setting. I'll likely change to nested classes when I get the chance. This will also make implementing config for verify and log reports easier.

Reply:
Updated to using nested classes